loadtest/run.go

```diff
@@ -82,82 +114,89 @@ func RunTest(test *TestRun) error {
 	waitMonitors.Add(1)
 	go ProcessClientRoundTripReports(clientTimingStats, clientTimingChannel3, clientTimingChannel, stopMonitors, &waitMonitors)
 
+	adminClient := getAdminClient(cfg.ConnectionConfiguration.ServerURL, cfg.ConnectionConfiguration.AdminEmail, cfg.ConnectionConfiguration.AdminPassword, nil)
+	if adminClient == nil {
+		return fmt.Errorf("Unable create admin client.")
+	}
+
+	if cfg.UserEntitiesConfiguration.EnableRequestTiming {
+		adminClient.HttpClient.Transport = NewTimedRoundTripper(clientTimingChannel)
+	}
+
 	numEntities := len(tokens)
-	entityNum := 0
-	entitiesToSkip := cfg.UserEntitiesConfiguration.EntityStartNum
-	for _, usertype := range test.UserEntities {
-		numEntitesToCreateForType := int(math.Floor((float64(usertype.Freq) / 100.0) * float64(numEntities)))
-		startEntity := 0
-		if numEntitesToCreateForType <= entitiesToSkip {
-			entitiesToSkip -= numEntitesToCreateForType
+	cmdlog.Infof("Starting %d entities from offset %d", numEntities, loadtestInstance.EntityStartNum)
+	for i := 0; i < numEntities; i++ {
```

Comment:
I found the previous strategy of iterating over the UserEntities and counting how many types to create difficult to adapt to this new pattern of coordinating among multiple load test instances. Now, just pick one at random according to the configured distribution.